torchci/pages/benchmark/[suite]/[compiler]/[[...page]].tsx

```diff
@@ -611,6 +619,56 @@ function ModelPanel({
                 const l = Number(v.l).toFixed(SCALE);
                 const r = Number(v.r).toFixed(SCALE);
 
+                if (lCommit === rCommit || l === r || v.r === 0) {
+                  return l;
+                } else {
+                  return `${r} → ${l}`;
+                }
+              },
+            },
+            {
+              field: "dynamo_peak_mem",
+              headerName: PEAK_MEMORY_USAGE_HEADER,
+              flex: 1,
+              cellClassName: (params: GridCellParams<any>) => {
+                const v = params.value;
+                if (v === undefined || v.r === 0) {
+                  return "";
+                }
+
+                const l = Number(v.l);
+                const r = Number(v.r);
+
+                if (lCommit === rCommit) {
+                  return "";
+                } else {
+                  if (l === 0 || l === r) {
+                    // 0 means the model isn't run at all
+                    return "";
+                  }
+
+                  // Decreasing more than x%
+                  if (r - l > PEAK_MEMORY_USAGE_RELATIVE_THRESHOLD * r) {
+                    return styles.ok;
+                  }
+
+                  // Increasing more than x%
+                  if (l - r > PEAK_MEMORY_USAGE_RELATIVE_THRESHOLD * r) {
+                    return styles.error;
```

Comment:
How does this error reflect itself in the dashboard?

Reply:
Atm, the regression is highlight as a failure on the dashboard.  Here is an example for moco/training/cudagraphs_dynamic/amp https://torchci-git-fork-huydhn-expose-max-cuda-mem-588d7e-fbopensource.vercel.app/benchmark/torchbench/inductor_dynamic?startTime=Thu%2C%2030%20Nov%202023%2022%3A04%3A30%20GMT&stopTime=Thu%2C%2007%20Dec%202023%2022%3A04%3A30%20GMT&granularity=hour&mode=training&dtype=amp&lBranch=main&lCommit=622688fab9fc6d20ff3475a8a0a1fdb6af9d837e&rBranch=main&rCommit=729ac7317a50a6a195b324cf6cefd748bf4f5498&model=moco